tests/test_rpc.py

```diff
@@ -1226,6 +1226,62 @@ def test_rbfpsbt_bump_fee(lianad, bitcoind):
         )
     )
 
+    # Now we test that if a tx is replaced externally to Liana and
+    # immediately confirmed, the spend_txid of the now-spent coin
+    # is set to the replacement txid.
+    # The replacement has to be done externally so that Liana doesn't
+    # see it until it is confirmed.
```

Comment:
It would still see it when it's included in the mempool.

Reply:
I should add to this comment that I stop and start lianad while the replacement is broadcast and confirmed to ensure that it isn't seen by Liana in the mempool.